front/src/reducers/osrdconf/osrdConfCommon/selectors.ts

```diff
@@ -59,12 +70,10 @@ const buildCommonConfSelectors = <ConfState extends OsrdConfState>(
       const pathSteps = getPathSteps(state);
       return pathSteps[pathSteps.length - 1];
     },
-    getViasV2: (state: RootState) => {
-      const pathSteps = getPathSteps(state);
-      pathSteps.shift();
-      pathSteps.pop();
-      return pathSteps;
-    },
+    /** To use this action, do useSelector(getViasV2()) */
+    getViasV2: () => viasSelector,
```

Comment:
I think you can do it like this, without warnings
```suggestion
    getViasV2: (state: RootState) => {
      const pathSteps = getPathSteps(state);
      return compact(pathSteps.slice(1, -1));
    },
```

Reply:
No I tried that first but then I get in the console the warning from the comment I put (and it can result to some white pages sometimes if we leave it)